src/mono/mono/mini/interp/transform.c

```diff
@@ -2982,6 +2966,10 @@ interp_inline_method (TransformData *td, MonoMethod *target_method, MonoMethodHe
 	int nargs = csignature->param_count + !!csignature->hasthis;
 	InterpInst *prev_last_ins;
 
+	if (header->code_size == 0)
+		/* IL stripped */
+		return FALSE;
+
```

Comment:
I don't think we should use `code_size == 0` as a sentinel for stripped IL.  In other places we use the absence of a `MonoMethodHeader*` to indicate that the method has been stripped.  That is consistent with other places in ECMA where methods have missing method bodies (such as `extern` methods).  I think the callers of `interp_inline_method` should properly handle the case case where `header` might be NULL and avoid calling `interp_inline_method` entirely.

Reply:
For these cases, I only trimmed away their function body. The headers are still there. So it won't work.

Reply:
ah ok, I misunderstood. Thanks!